src/prefect/utilities/templating.py

```diff
@@ -40,7 +55,10 @@ def find_placeholders(template: T) -> Set[Placeholder]:
         return set()
     if isinstance(template, str):
         result = PLACEHOLDER_CAPTURE_REGEX.findall(template)
-        return {Placeholder(*match) for match in result}
+        return {
+            Placeholder(match[0], match[1], determine_placeholder_type(match[1]))
```

Comment:
do these positional values have meaning? might be nicer for readability if you unpacked them `for item_one, item_two in result` (or whatever the names should be)